options/posix/generic/posix-file-io.cpp

```diff
@@ -15,6 +15,10 @@ int mem_file::close() {
 	return 0;
 }
 
+int mem_file::reopen(const char *, const char *) {
+	return -1;
```

Comment:
This is incorrect, an ``freopen()`` call on a memstream is valid, and should return a ``FILE*`` conforming to the input arguments. It should also flush the memstream and close it.

Reply:
Due to the large rework of both `FILE` and frigg internals needed for this, I have made this a panic for now.

Reply:
That's fine, we can add an issue for that and fix it later.